code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -146,36 +146,25 @@
 /obj/item/weapon/reagent_containers/pill/tramadol
 	name = "Tramadol pill"
 	desc = "A simple painkiller."
-	icon_state = "pill8"
+	icon_state = "pill11" //light blue round
 
 	New()
 		..()
 		reagents.add_reagent(TRAMADOL, 15)
 
-
-/obj/item/weapon/reagent_containers/pill/methylphenidate
```

Comment:
long since unused, the chemical these are supposed to spawn with doesn't even exist anymore which causes errors